src/i18n/en.js

```diff
@@ -234,10 +234,7 @@ export default {
     placeholder_email: `Your Email will be your Username`,
     placeholder_password: `Enter your secret key`,
     placeholder_phone: `9 11 2345-6789`,
-    privacy_policy_consent_HTML: `
-      I accept Doppler's
-      <a target="_blank" href="${urlPrivacyFromSignup}" ${patchForBlank}>Privacy Policy</a>.
-      `,
+    privacy_policy_consent_HTML: `I accept Doppler's <a target="_blank" href="${urlPrivacyFromSignup}" ${patchForBlank}>Privacy Policy</a>.`,
```

Comment:
In general, we try to have short lines in this file.

Reply:
i can change